tests/ode_test.py

```diff
@@ -232,6 +232,22 @@ def f(z):
 
     jax.grad(f)(jnp.ones(2))  # doesn't crash
 
+  @jtu.skip_on_devices("tpu", "gpu")
+  def test_complex_odeint(self):
+    # https://github.com/google/jax/issues/3986
+
+    def dy_dt(vals, t):
+      alpha, y = vals
+      return alpha, alpha * y
```

Comment:
Maybe try making `alpha` a "parameter" here rather than something that is explicitly integrated?
```python
def dy_dt(y, t, alpha):
  return alpha * y
```

Reply:
I have added this in the newest commit but it still produces the same error.